.eslintignore

```diff
@@ -4,6 +4,8 @@
 /app/*.cjs
 /**/ignore/**
 
+# Ignore generated camel model (need to fix generator to fix all the linting errors)
+/packages/hawtio/src/plugins/camel/model/camelModel.ts
```

Comment:
This is `eslintignore`. My intent is to ignore it when linting since the model is generated by
the mvn-generator. We can fix the generator but that's a little down the road at the moment. The current model flags up about ~150 errors in the linting.

Reply:
Added https://github.com/hawtio/hawtio-next/issues/95

Reply:
You are right. My bad, I thought it was .gitignore but it is .eslintignore ... Please ignore this comment. We won't even need to fix lint issues for the generated file.